homeassistant/components/teslemetry/switch.py

```diff
@@ -8,7 +8,7 @@
 from typing import Any
 
 from tesla_fleet_api.const import AutoSeat, Scope
-from tesla_fleet_api.teslemetry.vehicles import TeslemetryVehicle
+from tesla_fleet_api.teslemetry import Vehicle
```

Comment:
This looks like it dropped an import that is needed:

> Error: homeassistant/components/teslemetry/switch.py:41:24: F821 Undefined name `TeslemetryVehicle`
> Error: homeassistant/components/teslemetry/switch.py:42:25: F821 Undefined name `TeslemetryVehicle`

Reply:
Yep already fixed